project_task_project_mandatory/models/project_task.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+# © 2015-2017 Elico corp (https://www.elico-corp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models
+
+
+class ProjectTask(models.Model):
+    _inherit = 'project.task'
+
+    project_id = fields.Many2one(required=True)
```

Comment:
@dongshukai Problem with this if installed or upgrade need to make sure the project_id is not empty on table project_task otherwise will fail.
Suggest have pre_hook for new module
or migration folder for upgrading module